### ISSUE TITLE
Crash create_index: pymongo.errors.OperationFailure: cannot write to 'remains.system.indexes'

### ISSUE THREAD

**Issue by tonal** [twisted/txmongo#253]
versions:
txmongo 19.1.0
pymongo 3.9.0
MongoDB server version: 4.2.0

Code:
```python
    yield vendor_rrc.create_index(
      txmongo_sort(ASCENDING(['idprivider', 'series', 'idgood'])),
      unique=True)
```
Traceback:
```
...
  File "/home/user/lang/projects/remains/prices/pipelines.py", line 237, in open_spider
    unique=True)
  File "/home/user/lang/projects/remains/venv/lib/python3.7/site-packages/twisted/internet/defer.py", line 654, in _runCallbacks
    current.result = callback(current.result, *args, **kw)
  File "/home/user/lang/projects/remains/venv/lib/python3.7/site-packages/txmongo/protocol.py", line 459, in on_reply
    raise OperationFailure(err, code = code)
pymongo.errors.OperationFailure: cannot write to 'remains.system.indexes'
```

**Comment by IlyaSkriblovsky** [comment]
Sad to say, but probably nobody have tested TxMongo with MongoDB 4.2 yet.

I'm likely going to had some time to fix 4.2 (or at least 4.0) compatibility issues during by work hours soon. But your pull requests are welcome too :)

**Comment by IlyaSkriblovsky** [comment]
Fixed by #257
